### ISSUE TITLE
Do you have any plan to open the source of xjsfl.dll?

### ISSUE THREAD

**Issue by jinkim-ea** [davestewart/xjsfl#16]
Hi,

I'm interested in using xjsfl and itself. I'm very curious of the internal of xjsfl.dll. Do you have any plan to open the source of xjsfl.dll?

Thanks,
JinSan

**Comment by fairsquare** [comment]
Hi JinSan,

The .dll is xJSFL's biggest secret...

You know why?

It's a cheat!

I discovered early on in development that some of the other plugins, when written to from JSFL would retain values for the entire session.

I did some research on how to create my own .dll but it was beyond my reach, so in a flash of insight, I copied and renamed the Macromedia example tool `blah.dll` to `xjsfl.dll`... and it just worked!

So, no magic, just smoke and mirrors :P

**Comment by davestewart** [comment]
Hi JinSan,

The .dll is xJSFL's biggest secret...

You know why?

It's a cheat!

I discovered early on in development that some of the other plugins, when written to from JSFL would retain values for the entire session.

I did some research on how to create my own .dll but it was beyond my reach, so in a flash of insight, I copied and renamed the Macromedia example tool `blah.dll` to `xjsfl.dll`... 

...and it just worked!

So, no magic, just smoke and mirrors 😜 

I've never seen the internals, and have no idea what is in there!
